src/Ombi/ClientApp/src/app/requests-list/components/_shared-card-grid.scss

```
@@ -45,12 +45,151 @@
+    &.active {
+      background: $accent-dark;
+      color: #111;
+    }
+  }
+}
+
+// Dropdown menus for the toolbar pills - rendered in the CDK overlay,
+// so this cannot be scoped to :host
+/* stylelint-disable-next-line selector-pseudo-element-no-unknown */
+::ng-deep .filter-menu.mat-mdc-menu-panel, /* NOSONAR */
+::ng-deep .filter-menu.mat-menu-panel { /* NOSONAR */
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)